setup.cfg

```diff
@@ -9,7 +9,13 @@ max-line-length = 120
 
 [flake8]
 max-line-length = 120
-ignore = F401,E402,F403,W503,W504,F821
+ignore = E203,E402,F403,W503,W504,F821
+per-file-ignores =
+    __init__.py: F401, F403, F405
```

Comment:
F403 seems to be a global ignore above, maybe we can remove it?

Reply:
F403 will be raised if `flake8` finds `from foo import *`. We should remove this from the global ignores and only allow this in the files below.

Reply:
There are none besides in the files in that list. Thus, I think we should go forward with my plan.

Reply:
I agree with the plan, but F403 is still part of the global `ignore` list right now

Reply:
Sorry, misunderstood your intent.